script/src/types.rs

```diff
@@ -350,12 +350,22 @@ pub enum VmState {
     WaitForRead(ReadState),
 }
 
+#[derive(Clone, Debug)]
+pub struct ExecV2Args {
+    pub data_piece_id: DataPieceId,
+    pub offset: u64,
+    pub length: u64,
```

Comment:
Notice both `ExecV2Args` and `SpawnArgs` contain `data_piece_id`, `offset` and `length`. All 3 are required to locate a program. We should make them a struct to distinguish them from other parameters.